src/zac/demo/mor/tests/tests.py

```diff
@@ -0,0 +1,105 @@
+import collections
+import json
+import requests
+from django_webtest import WebTest
+from time import sleep
+
+from django.conf import settings
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import gettext
+
+from ...models import SiteConfiguration
+
+from .factories import *
```

Comment:
don't `import *`, use explicit imports

it confuses editors, it's prone to import shadowing and pollutes namespaces with stuff that you don't use